CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added custom certificates documentation
```

Comment:
You should not edit the changelog manually, please follow this https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md?plain=1#L8C1-L15C4

Reply:
OK,I will change it